libraries/bot-connector/src/test/java/com/microsoft/bot/connector/authentication/ParameterizedBotFrameworkAuthenticationTest.java

```diff
@@ -0,0 +1,205 @@
+
+        AuthenticationConfiguration authenticationConfiguration = new AuthenticationConfiguration();
+        ParameterizedBotFrameworkAuthentication parameterizedBotFrameworkAuthentication = new ParameterizedBotFrameworkAuthentication(true, "test@example.org", "test@example.org", "ABC123",
+            "https://example.org/example", "test@example.org", "test@example.org", "42", passwordServiceClientCredentialFactory,
+            authenticationConfiguration, new OkHttpClient());
+        ClaimsIdentity claims = parameterizedBotFrameworkAuthentication
+            .authenticateChannelRequest(null).join();
+
+        assertNotNull(claims);
+    }
+
+    @Test
+    public void authenticateChannelRequest_withAuthenticationDisabled_shouldThrowAnError(){
+        PasswordServiceClientCredentialFactory passwordServiceClientCredentialFactory = Mockito.mock(PasswordServiceClientCredentialFactory.class);
+        Mockito.when(passwordServiceClientCredentialFactory.isAuthenticationDisabled()).thenReturn(CompletableFuture.completedFuture(false));
+        AuthenticationConfiguration authenticationConfiguration = new AuthenticationConfiguration();
+
+        ParameterizedBotFrameworkAuthentication parameterizedBotFrameworkAuthentication = new ParameterizedBotFrameworkAuthentication(true, "test@example.org", "test@example.org", "ABC123",
+            "https://example.org/example", "test@example.org", "test@example.org", "42", passwordServiceClientCredentialFactory,
+            authenticationConfiguration, new OkHttpClient());
+
+        assertThrows(CompletionException.class, () -> {
+            parameterizedBotFrameworkAuthentication.authenticateChannelRequest(null).join();;
+        });
+    }
+
+    @Test
+    public void authenticateRequest_withAuthenticationEnabled_shouldNotBeNull(){
+        PasswordServiceClientCredentialFactory passwordServiceClientCredentialFactory = new PasswordServiceClientCredentialFactory("42",
+            "test");
+
+        AuthenticationConfiguration authenticationConfiguration = new AuthenticationConfiguration();
+        ParameterizedBotFrameworkAuthentication parameterizedBotFrameworkAuthentication = new ParameterizedBotFrameworkAuthentication(true, "test@example.org", "test@example.org", "ABC123",
+            "https://example.org/example", "test@example.org", "test@example.org", "42", passwordServiceClientCredentialFactory,
+            authenticationConfiguration, new OkHttpClient());
+
+        Activity activity = Activity.createConversationUpdateActivity();
+        activity.setChannelId(Channels.EMULATOR);
+        ChannelAccount channelAccount = new ChannelAccount();
+        RoleTypes roleTypes = RoleTypes.SKILL;
+        channelAccount.setRole(roleTypes);
+        activity.setRecipient(channelAccount);
+
+        AuthenticateRequestResult result = parameterizedBotFrameworkAuthentication.authenticateRequest(activity,
+                null).join();
+        assertNotNull(result);
```

Comment:
Same here. Can we add assertions to validate the content of  `result` aside from not being `null`?